### PR TITLE
rdrview: unstable-2020-12-22 -> unstable-2021-05-30

### PR DESCRIPTION
**Pull request by L-as**
###### Motivation for this change

Updated it.
Added man page to output too.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by djanatyn — APPROVED** [review]
Thanks for this change! Looks good to me:
```
❯ nix shell github:L-as/nixpkgs/647d177240ec1b464312763dfb202ccdc8249b1c#rdrview
❯ man rdrview | head
rdrview(1)                  General Commands Manual                 rdrview(1)

NAME
       rdrview - extract readable content from a webpage

SYNOPSIS
       rdrview  [-v] [-u base-url] [-E encoding] [-A user-agent] [-T template]
       [-c|-H|-M|-B browser] [path|url]

DESCRIPTION

❯ which rdrview
/nix/store/89rlrsflirg6p69p577bwv2jl7dg090r-rdrview/bin/rdrview
❯ rdrview --version
rdrview version 0.1
```

File: pkgs/tools/networking/rdrview/default.nix [hunk 1]
```diff
@@ -2,21 +2,25 @@
...
+    rev = "444ce3d6efd8989cd6ecfdc0560071b20e622636";
+    sha256 = "02VC8r8PdcAfMYB0/NtbPnhsWatpLQc4mW4TmSE1+zk=";
   };
 
   buildInputs = [ libxml2 curl libseccomp ];
 
   installPhase = ''
     install -Dm755 rdrview -t $out/bin
+    mkdir -p $out/share/man/man1
+    gzip -c rdrview.1 > $out/share/man/man1/rdrview.1.gz
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/rdrview/default.nix:19]: Compression is done automatically and please use installManPage.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 134663` at 647d1772 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>rdrview</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/networking/rdrview/default.nix:16:3:
```
   |
16 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/name-and-version.md">name-and-version</a>

Did you mean to pass `pname` instead of `name` to `mkDerivation`?

Near pkgs/tools/networking/rdrview/default.nix:4:3:
```
  |
4 |   name = "rdrview";
  |   ^
```

Near pkgs/tools/networking/rdrview/default.nix:5:3:
```
  |
5 |   version = "unstable-2021-05-30";
  |   ^
```
</li>
  </ul>
</details>

**Comment by L-as** [comment]
Should be fine now.

**Pull request merged**